mbuild/compound.py

```diff
@@ -35,8 +35,14 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
     Parameters
     ----------
     filename : str
-    relative_to_module :
+        Name of the file from which to load atom and bond information. Files
+        are read using the mdtraj package. Please refer to http://mdtraj.org/
+        1.8.0/load_functions.html for supported formats.
+    relative_to_module : str, optional
+        Path to the directory of the file. Default is the current working
```

Comment:
This isn't quite correct. The actual code will look in `sys.modules` and find the path of the module used here. 

I think you'll want to a) specify that the default is None and that b) show a typical use case like in the library compounds.